snap/gui/coronabar.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Version=1.0
```

Comment:
again.. Under `ubuntu directory`

Reply:
Same thing.
To automatically make the snap on snapcraft.io the current structure must be kept. It's that or having to manually produce a snap on every future code change...